src/main/java/com/pileproject/drivecommand/machine/device/input/simulation/SimulatedColorSensor.java

```diff
@@ -0,0 +1,38 @@
+package com.pileproject.drivecommand.machine.device.input.simulation;
```

Comment:
So, at first, let me confirm options. They are; 

- `com.pileproject.drivecommand.machine.device.input.mock` (as followed the naming discussion)
- `com.pileproject.drivecommand.mock.machine.device.input`

right?